src/main/java/spireMapOverhaul/zones/volatileGrounds/VolatileGrounds.java

```diff
@@ -0,0 +1,79 @@
+package spireMapOverhaul.zones.volatileGrounds;
+
+import com.badlogic.gdx.graphics.Color;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.monsters.MonsterGroup;
+import com.megacrit.cardcrawl.monsters.beyond.Exploder;
+import com.megacrit.cardcrawl.monsters.exordium.LouseNormal;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.abstracts.AbstractZone;
+import spireMapOverhaul.zoneInterfaces.EncounterModifyingZone;
+import spireMapOverhaul.zones.volatileGrounds.monsters.Eruptor;
+import spireMapOverhaul.zones.volatileGrounds.monsters.HeatBlister;
+import spireMapOverhaul.zones.volatileGrounds.monsters.SunStoneShard;
+import spireMapOverhaul.zones.volatileGrounds.monsters.UnstableSunstone;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class VolatileGrounds extends AbstractZone implements EncounterModifyingZone {
+    private static final String SUNSTONE_ELITE = SpireAnniversary6Mod.makeID("SUNSTONE_ELITE");
+    private static final String EXPLOSIVE_SHAPES = SpireAnniversary6Mod.makeID("EXPLOSIVE_SHAPES");
+    private static final String HEAT_BLISTER = SpireAnniversary6Mod.makeID("HEAT_BLISTER");
+    
+    public VolatileGrounds() {
+        super("VolatileGrounds", Icons.ELITE, Icons.MONSTER);
+        this.width = 1;
+        this.height = 4;
+        System.out.println("Volatile Grounds" + name + " " + width + "x" + height);
+    }
+    
+    @Override
+    public AbstractZone copy() {
+        return new VolatileGrounds();
+    }
+    
+    @Override
+    public Color getColor() {
+        return Color.ORANGE.cpy();
+    }
+    
+    @Override
+    public boolean canSpawn() {
+        return isAct(3);
+    }
+    
+    @Override
+    public List<ZoneEncounter> getNormalEncounters() {
+        return Arrays.asList(
+                new ZoneEncounter(EXPLOSIVE_SHAPES, 3, () -> new MonsterGroup(
+                        new AbstractMonster[]{
```

Comment:
The enemies in this fight are a bit too close together. Need to space them out a bit. Try 180 or 200 pixels between each enemy (instead of the current 160).
![image](https://github.com/erasels/SpireMapOverhaul/assets/62083413/f6685996-1776-46f6-81a2-f05941adf4ed)